extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/AuthenticationProviderService.java

```diff
@@ -306,17 +306,42 @@ public LDAPAuthenticatedUser authenticateUser(Credentials credentials)
         }
 
     }
+    /**
+     * Returns parameter current ldap domain token generated from user credentials
+     * If no multiple LDAP are configured on GUACAMOLE_HOME such ldap-servers.yaml,
+     * a null is returned.
+     *
+     * @param credentials
+     *     The credentials to use for authentication.
+     *
+     * @return
+     *     Domain name by splitting logged username(domain/username) when multiple LDAP configuration is available
+     *     or null if no such configuration
+     */
+    private String getDomainToken(Credentials credentials) {
+        String ldapDomainName = null;
+        if (credentials.getUsername().contains("\\")) {
+            ldapDomainName = credentials.getUsername().split("\\\\")[0];
+        } else if (credentials.getUsername().contains("@")) {
```

Comment:
Please follow established style guidelines - in particular, don't "cuddle" the tags.

Reply:
@necouchman Could you give us more context, couldn't figure out what your comments is. thanks.

Reply:
@josnabattula Sure:
`} else if (something else) {`

This is known as "cuddling" tags. Code style for this project is:
```
}
else if (something else) {
```

Where the `}` is not on the same line as the `else if` statement.

Reply:
Thanks for response, figured out :)